src/ports/postgres/modules/linalg/svd.py_in

```diff
@@ -1070,16 +1091,17 @@ def lanczos_bidiagonalize_sparse(schema_madlib, source_table,
     if nIterations is None:
         nIterations = min(col_dim, MAX_LANCZOS_STEPS)
     elif nIterations < k or nIterations > col_dim:
-        plpy.error("SVD error: Number of Lanczos iterations should be in the"
+        plpy.error("SVD error: nIterations should be in the"
```

Comment:
Should this be consistant with what we have on our doc, as n_iterations instead of nIterations?

Reply:
I have changed it as per the discussion in Jira.